tests/Makefile.in

```diff
@@ -187,7 +187,8 @@ am__unit_tests_dbg_SOURCES_DIST = driver.C test_comm.h \
 	fe/fe_l2_lagrange_test.C fe/fe_lagrange_test.C \
 	fe/fe_monomial_test.C fe/fe_szabab_test.C fe/fe_test.h \
 	fe/fe_xyz_test.C geom/elem_test.C geom/node_test.C \
-	geom/point_test.C geom/point_test.h mesh/all_tri.C \
+	geom/point_test.C geom/point_test.h \
+	geom/which_node_am_i_test.C mesh/all_tri.C \
 	mesh/boundary_mesh.C mesh/boundary_info.C mesh/checkpoint.C \
```

Comment:
Ugh, yeah. I figured it was OK in this instance since it was only a 1 line change in Makefile.am, but I can split them up.